src/packages/colors/tsconfig.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "../../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "lib",
+    "composite": true
+  },
+  "include": ["src"]
```

this is missing the index ts file and is not compiling it

```suggestion
  "include": ["src", "index.ts"]
```